tests/python/pants_test/subsystem/test_subsystem.py

```diff
@@ -266,3 +275,17 @@ def subsystem_dependencies(cls):
 
     with self.assertRaises(SubsystemClientMixin.CycleException):
       list(SubsystemB.subsystem_closure_iter())
+
+  def test_get_streaming_workunit_callbacks(self):
+    import_str = "pants_test.subsystem.test_subsystem.WorkunitSubscriptableSubsystem"
+    callables_list = Subsystem.get_streaming_workunit_callbacks([import_str])
+    assert len(callables_list) == 1
+
+  def test_streaming_workunit_callbacks_with_invalid_subsystem(self):
```

Comment:
Would be good to test the other failure points like valid module but no class defined.

Reply:
+1 to the above test suggestion!